dojo/models.py

```diff
@@ -1196,7 +1196,7 @@ def clean(self):
                 self.userinfo = None
 
         if self.host:
-            if not re.match(r'^[A-Za-z0-9][A-Za-z0-9_\.\-\+]+$', self.host):
+            if not re.match(r'^[A-Za-z0-9_\-\+][A-Za-z0-9_\.\-\+]+$', self.host):
```

Comment:
Should we remove the underscore? We are validating the host name here @kiblik

Reply:
@damiencarol, no. Underscore was added there to accept also hosts like `_invalid._host.com`

Reply:
ok. so permissive way. :) I'm good with that